apiserver/internal/handlers/resources/download/downloader.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package download
```

Comment:
Is this `download` or `downloader`, the two terms are being used interchangeably with package, folder and file names. Pick one.

Reply:
The package is `download` and which provides a `Downloader` with a `Download` method. I've renamed the file that was called `downloader`, that was a mistake.